test/unit/features/directives/bind.spec.js

```diff
@@ -234,7 +234,7 @@ describe('Directive v-bind', () => {
       template: `<test v-bind="test" data-foo="foo" dataBar="bar"/>`,
       components: {
         test: {
-          template: '<div :data-foo="dataFoo" :data-bar="dataBar"></div>',
+          template: '<div>{{ dataFoo }} {{ dataBar }}</div>',
```

Comment:
The old spec passed because `data-foo` fell through as an attribute and coincidentally appeared in the final output. Using plain text instead to ensure we got the correct value.